wallet/address.js

```diff
@@ -22,7 +22,7 @@ function addressHash(input) {
   const context = blake2.blake2bInit(28) // blake2b-224
   blake2.blake2bUpdate(context, firstHash)
 
-  return new Buffer(blake2.blake2bFinal(context)).toString('hex')
```

Comment:
I hope this is covered by the tests

Reply:
yes. The address encoding test would fail if this returned wrong output

Reply:
are thore bugs or features: ? 
- `1.` is treated as invalid num (any num + `.` 
- if user tires to mess up / del address, the amount validations will  disappear, and red total will be actualized no more until a valid address is typed

Reply:
@git-jano "1." passes the validation - that's intended behavior. The fee staying red and submit button being disabled  while the address is missing/invalid is also intended.

Reply:
I was wondering why the "1." actually passes the validation despite the regex and I found out that the HTML5 input element sanitizes the input (at least in chrome and firefox) and strips the comma if it is not followed by any number, therefore the value "1." is transformed into "1" before being fed to Javascript. Finally I decided to make the "send-amount" input a standard input and handle the validation just in javascript to avoid such confusion and to have absolute control over all the cases.